### PR TITLE
161874264 transport operator info toggles

### PR DESCRIPTION
**Pull request by philosotractor**
# Changed
* ui: info-toggle default state true i.e. expanded
* ui: form field support for info-toggle component
* language: initial localisation for operator ytj help info toggles
* views: transport-operator ytj info-toggles

### CODE REVIEW

File: ote/resources/public/language/en.edn [hunk 1]
```diff
@@ -636,14 +636,30 @@ You can also draw the operating area or point to the map with drawing tools. You
...
   :basic-info-tooltip "Fill in to Basic information -section information about the company, community or organization you are representing."
   :contact-types-tooltip "Phone and email -section information is meant for utilisers of NAP-interfaces - not to the end users (customers) of your transport service."
   :ckan-description-tooltip "The purpose of the transport operator description is to give NAP utilisers an idea about what kind of transport operator they are dealing with. E.g. Example Ltd. is a company conducting scheduled transport and charter traffic in Southern Finland."
+  :help-ytj-integration-desc "The service provider information is received from the _Business Information System (BIS) on the basis of the Business ID. If the information is incorrect or incomplete, you can make changes to it in the _BIS service_. Changes made in the BIS service (e.g. changing the street address or registering a new auxiliary trade name) will appear on this page after they are registered in the BIS register."
```
Comment (rckopteri) [inline ote/resources/public/language/en.edn]: Why _Business  and not Business

File: ote/resources/public/language/en.edn [hunk 2]
```diff
@@ -636,14 +636,30 @@ You can also draw the operating area or point to the map with drawing tools. You
...
+  :help-operator-how-delete (:md "To remove the service provider information from the NAP service, click the **Delete service provider** -button. Deleting the service provider is possible only when none of the service providers created (for the business name or for the auxiliary names) under the Business ID have any published services in the NAP service. If you want to delete the service provider, please, first delete all the transport services on the **Own transport services** page.")
+  :help-about-ytj-link-desc "Lue lisää Yritys- ja yhteisötietojärjestelmästä (YTJ)"
```
Comment (rckopteri) [inline ote/resources/public/language/en.edn]: Read more about Business Information System (BIS)

File: ote/src/cljs/ote/views/transport_operator_ytj.cljs [hunk 3]
```diff
@@ -145,19 +161,28 @@
...
        :max-length 128
        :h-style    :h3}
 
+      (when response-ok?
+        {:name          :help-operator-contact-details
+         :type          :info-toggle
+         :label         (tr [:common-texts :instructions])
+         :body          [:div
+                         (tr [:organization-page :help-operator-contact-entry])
+                         [uicommon/extended-help-link "https://www.ytj.fi/index/ilmoittaminen/muutosilmoitus.html" "Tietojen muutosilmoitus Yritys- ja yhteisötietojärjestelmässä (YTJ)"]]
```
Comment (rckopteri) [inline ote/src/cljs/ote/views/transport_operator_ytj.cljs]: You need to pass this url based on users language. In english: https://www.ytj.fi/en/index/notifications/notificationsofchanges.html

File: ote/src/cljs/ote/views/transport_operator_ytj.cljs [hunk 4]
```diff
@@ -236,35 +261,35 @@
...
+                            [:br]
+                            [:div [:h3 (tr [:dialog :delete-transport-operator :title-base-view])]]
+                            [info/info-toggle (tr [:common-texts :instructions]) (tr [:organization-page :help-operator-how-delete]) true]
+                            [buttons/save {:on-click #(e! (to/->ToggleTransportOperatorDeleteDialog))
+                                           :disabled (if (::t-operator/id data) false true)}
+                             (tr [:buttons :delete-operator])]]))])})
 
 (defn operator-ytj [e! {operator :transport-operator :as state}]
-  ;(e! (to/->EditTransportOperator (get-in state [:params :id])))
   (fn [e! {operator :transport-operator :as state}]
```
Comment (philosotractor) [inline ote/src/cljs/ote/views/transport_operator_ytj.cljs]: Old code, but this could be refactored out

**Pull request merged**